lib/generators/draper/install_generator.rb

```diff
@@ -1,7 +1,7 @@
 module Draper
   module Generators
     class InstallGenerator < Rails::Generators::Base
-      source_root File.expand_path('../templates', __FILE__)
+      source_root File.expand_path("templates", __dir__)
```

Comment:
Is there a reason for this change? I understand that it will do the same thing. However, when generating a generator, Rails still outputs the original. I'm curious what the inspiration was or why you prefer this?